daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -279,6 +279,16 @@ sealed abstract class HasTxNodes[Nid, +Cid] {
 
   def roots: ImmArray[Nid]
 
+  // TODO https://github.com/digital-asset/daml/issues/8020
+  //  check this make seen w.r.t. the ledger model
+  /** The informees of a all transaction.
+    */
+  def informees: Set[Ref.Party] =
```

Comment:
Wondering if I should make this a `lazy val`

Reply:
I’d probably go for the lazy val but I don’t feel particularly strongly about this.